libs/barista-components/filter-field/index.ts

```diff
@@ -22,7 +22,11 @@ export * from './src/filter-field-range/filter-field-range-trigger';
 export * from './src/filter-field-data-source';
 export * from './src/filter-field-default-data-source';
 export * from './src/filter-field-errors';
-export { applyDtOptionIds, DELIMITER } from './src/filter-field-util';
+export {
+  applyDtOptionIds,
+  DELIMITER,
+  defaultTagDataForFilterValuesParser,
```

Comment:
@thomaspink If we export them now, do we need to prefix or at least annotate them to be from the library?

```suggestion
  applyDtOptionIds,
  DT_FILTER_FIELD_DELIMITER,
  dtDefaultTagDataForFilterValuesParser,
```

Reply:
would suggest so but not that important for me